### DISCUSSION TITLE
Billing _class for pharmacy claims

### DISCUSSION THREAD

**Discussion by CH-PikeSowle** [CMSgov/price-transparency-guide#416] [Q&A]
The Allowed Amount Object schema contains a requirement to include the billing_class.  The field allows two possible values, which we don't think apply to pharmacy claims.  Should the billing-class be optional for pharmacy claims?

**Comment by shaselton-usds** [comment]
Trying to understand this better -- in what situation would a pharmacy claim not fall into either a professional or institutional billing class?

**Reply by CH-PikeSowle** [reply]
Hi Scott - The concept of institutional vs professional does not exist in RX claims.  This is what I am hearing from our claims experts.

**Reply by CKreklow** [reply]
Are you attempting to report pricing that would apply to point-of-sale retail pharmacy claims (NCPDP)?  It is my understanding that those were put on hold indefinitely in the delayed enforcement guidance, and it's only necessary to report "medical" drug pricing that should fall into the traditional professional / institutional categories.

**Reply by bryanbriegel** [reply]
in the Aug 20, 2021 FAQs posted by the DOL:  "...as an exercise of enforcement discretion, the Departments will defer enforcement of the requirement in the TiC Final Rules that plans and issuers must publish machine-readable file related to prescription drugs while it considers, through notice-and-comment rulemaking, whether the prescription drug machine-readable file requirement remains appropriate."  and in the Final Rule the Departments note: _"To reduce the complexity and burden of including prescription drug information in the negotiated rate machine-readable file, the final rules require a third file that will include pricing information for prescription drugs"_
